dids/src/main/kotlin/web5/sdk/dids/DidIon.kt

```diff
@@ -446,37 +451,22 @@ public sealed class DidIonManager(
       ),
       KeyAliases(
         updateKeyAlias = updateKeyAlias,
-        verificationKeyAlias = verificationKeyAlias,
+        verificationKeyAlias = publicKeysWithAlias.mapNotNull { it.first },
         recoveryKeyAlias = recoveryKeyAlias
       )
     )
   }
 
-  private fun getVerificationMethodIdOrDefault(options: VerificationPublicKeyOption?) =
-    when (options?.verificationMethodId) {
-      null -> UUID.randomUUID().toString()
-      else -> {
-        validateId(options.verificationMethodId!!)
-        options.verificationMethodId!!
-      }
-    }
-
-  private fun getVerificationPublicKeyOrDefault(options: VerificationPublicKeyOption?, keyManager: KeyManager) =
-    if (options?.verificationPublicKey == null) {
-      val verificationMethodId = getVerificationMethodIdOrDefault(options)
-      val alias = keyManager.generatePrivateKey(JWSAlgorithm.ES256K)
-      val verificationJwk = keyManager.getPublicKey(alias)
-      Pair(
-        PublicKey(
-          id = verificationMethodId,
-          type = "JsonWebKey2020",
-          publicKeyJwk = verificationJwk,
-          purposes = listOf(PublicKeyPurpose.AUTHENTICATION, PublicKeyPurpose.ASSERTION_METHOD),
-        ),
-        alias
-      )
+  private fun maybeGeneratePublicKeysToAdd(options: CommonOptions?, keyManager: KeyManager) =
```

Comment:
nit: I know `maybe` in function names is totally a java thing buuut it's a pet peeve of mine 😢

Reply:
Renamed to `publicKeysWithAliasesToAdd`. Maybe that's better?